packages/react-button/package.json

```diff
@@ -29,6 +29,7 @@
     "@fluentui/babel-make-styles": "9.0.0-beta.2",
     "@fluentui/eslint-plugin": "*",
     "@fluentui/jest-serializer-make-styles": "9.0.0-beta.2",
+    "@fluentui/react-components": "9.0.0-beta.3",
```

Comment:
Do not use these as depedencies, as we discussed storybook depedencies are not really dependencies for the package

```suggestion
```